packages/dependency-extraction-webpack-plugin/lib/index.js

```diff
@@ -97,6 +97,16 @@ class DependencyExtractionWebpackPlugin {
 		}
 
 		if ( externalRequest ) {
+			// Special case for modules.
+			// Since `@wordpress/polyfill` is a script and not a module, we need
+			// to exclude it from the list of externalised deps.
+			if (
+				this.useModules &&
+				externalRequest.includes( '@wordpress/polyfill' )
+			) {
+				return callback( null, '' );
```

Comment:
Is this attempting to ignore the module during when externalizing?

Reply:
Yes, that's correct! I sneaked this change in today to see if it fixed the tests. If it works, I'll see if I can simplify it later 🙂

Reply:
It looks like it would be necessary to be included somehow for `@wordpress/interactivity-router`.

Reply:
Right, I think this exposed the fact that while `@wordpress/interactivity-router` doesn't currently depend on the polyfills, it probably should, if we want it to work correctly in older browsers like Chrome 109. Although there isn't really a good mechanism to make that happen, as far as I know 🤔